manifests/n/NeumetrixLimited/HttpWatchBasic14/0Setup/14.0.16/NeumetrixLimited.HttpWatchBasic14.0Setup.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: NeumetrixLimited.HttpWatchBasic14.0Setup

```

```suggestion
PackageIdentifier: NeumetrixLimited.HttpWatchBasic.14
```
Package folder will need to be updated when identifier is changed